integration/airflow/openlineage/airflow/macros.py

```diff
@@ -1,56 +1,93 @@
 # Copyright 2018-2024 contributors to the OpenLineage project
 # SPDX-License-Identifier: Apache-2.0
 
+from __future__ import annotations
+
 import typing
 
 from openlineage.airflow.adapter import _DAG_NAMESPACE, OpenLineageAdapter
 from openlineage.airflow.utils import get_job_name
 
 if typing.TYPE_CHECKING:
-    from airflow.models import TaskInstance
+    from airflow.models import BaseOperator, TaskInstance
+
+
+def lineage_job_namespace():
+    """
+    Macro function which returns Airflow OpenLineage namespace.
+    Invoke as a jinja template, e.g.
+
+    PythonOperator(
+        task_id='render_template',
+        python_callable=my_task_function,
+        op_args=['{{ lineage_job_namespace() }}'], # invoke macro
+        provide_context=False,
+        dag=dag
+    )
+    """
+    return _DAG_NAMESPACE
 
 
-def lineage_run_id(task_instance: "TaskInstance"):
+def lineage_job_name(task: BaseOperator | TaskInstance):
```

Comment:
It's a minor but I'd rather hint and use task instance only. This would require changes in docs too.

Reply:
Done